topology/Makefile.am

```diff
@@ -5,4 +5,4 @@
 #
 
 xmldir = ${pkgdatadir}/xml
-dist_xml_DATA = g5.48xl-topo.xml p4d-24xl-topo.xml p4de-24xl-topo.xml p5.48xl-topo.xml
```

Comment:
Since we have it, let's ship the file for this release, just in case.

Reply:
Added it again.